test/models/page.test.js

```diff
@@ -0,0 +1,145 @@
+var chai = require('chai')
+  , expect = chai.expect
+  , sinon = require('sinon')
+  , sinonChai = require('sinon-chai')
+  , proxyquire = require('proxyquire')
+  ;
+chai.use(sinonChai);
+
+describe('Page', function () {
+  var conn
+    , app = new express()
+    , models = proxyquire(MODEL_DIR, {mongoose: mongoose})(app)
+    , Page = proxyquire(MODEL_DIR + '/page.js', {mongoose: mongoose})(app, models)
+    , User = proxyquire(MODEL_DIR + '/user.js', {mongoose: mongoose})(app, models)
+    ;
+
+  if (!mongoUri) {
+    return;
+  }
+
+  before(function (done) {
+    conn = mongoose.createConnection(mongoUri, function(err) {
+      if (err) {
+        done();
+      }
+
+      Page = conn.model('Page');
+      User = conn.model('User');
+
+      var fixture = [
+        {
+          path: '/user/anonymous/memo',
+          grant: Page.GRANT_RESTRICTED,
+          grantedUsers: []
+        },
+        {
+          path: '/grant/public',
+          grant: Page.GRANT_PUBLIC
+        },
+        {
+          path: '/grant/restricted',
+          grant: Page.GRANT_RESTRICTED
+        },
+        {
+          path: '/grant/specified',
+          grant: Page.GRANT_SPECIFIED
+        },
+        {
+          path: '/grant/owner',
+          grant: Page.GRANT_OWNER
+        }
+      ];
+      var userFixture = [
+        {userId: 'anonymous', email: 'anonymous@gmail.com'}
+      ];
+
+      testDBUtil.generateFixture(conn, 'Page', fixture, function() {});
+      testDBUtil.generateFixture(conn, 'User', userFixture, done);
```

Comment:
It would be better if we could make `testDBUtil` more general to add multiple fixtures.